### PR TITLE
Fix error on IE/Edge

### PR DESCRIPTION
**Pull request by Arthelon**
Temporary fix for #55 . Removes the PrismJS import in `CodeEditor.svelte` which disables syntax highlighting and also resolves the error in Edge/IE.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +18,7 @@
     "lint:css": "stylelint \"src/**/*.{svelte,css}\""
   },
   "dependencies": {
+    "@babel/runtime": "7.10.1",
```
Comment (kiootic) [inline package.json:21]: Why is this needed?

Comment (Arthelon) [reply package.json:21]: It is a production dependency that replaces `@babel/plugin-transform-runtime`. https://babeljs.io/docs/en/babel-plugin-transform-runtime

Comment (kiootic) [reply package.json:21]: Oh I see now, you are updating the existing `babel-runtime` dependency.

**Review by kiootic — APPROVED** [review]
